### PR TITLE
Fixing Process parameters

### PR DESCRIPTION
**Pull request by YaronFruchtmann**

### CODE REVIEW

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml [hunk 1]
```diff
@@ -79,16 +79,19 @@ ParserQuery: |
     // --------------------------------------------------------------------------------------
     | extend
       EventDetails = todynamic(EventDetails)
+    | where EventDetails.EventType != 'EXIT'
```
Comment (oshezaf) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml:82]: Filtering has to be done earlier as to not work on process terminate so much. As you did in terminate....

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml (hunk 1 above)
Comment (YaronFruchtmann) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml:82]: with terminate I require inclusion, and later verified it is included in the correct place.
Here the requirement is NOT to include, so I am risking omitting cases where "EXIT" appears elsewhere

Comment (oshezaf) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml:82]: Not if you check for the entire fragment that includes EXIT, after all it is JSON and not free text.

Comment (YaronFruchtmann) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml:82]: After offline discussion, will not change now. 
TODO: in prefilter using the full key value segment of the EXIT value

**Review by oshezaf — APPROVED** [review]

**Pull request merged**